hitch/maintainer.md

```diff
@@ -0,0 +1 @@
+[the Hitch Docker Community](%%GITHUB-REPO%%)
```

Comment:
Did you want this to be something stronger, given it's officially maintained by/with the Hitch project, or are you happy with this wording? (just to be sure :+1:)

Reply:
I'd say it's fine, unless there's another angle than glory and recognition I haven't seen :-)